src/3dsFormatToAntsBinaryBuffer.js

```diff
@@ -43,7 +43,7 @@ function encodeVertex(x, y, z){
 
     returns a Buffer encoded as :
     
-    object : | nbVertices(2) | v1(4) | v2(4) | ... | vn(4) | nbFaces(2) | f1(6) | f2(6) | ... | fn(6) |
+    object : | nbVertices(2) | v1(4) | v2(4) | ... | vn(4) | nbFaces(2) | f1(6) | f2(6) | ... | fn(6) | meshType(2)
```

Comment:
I have no opinion to be honest. It was just a drive-by comment.
As long as it's clearly documented (maybe the comments we have now are enough), I don't think it makes a difference.